transaction/core/src/lib.rs

```diff
@@ -63,6 +63,9 @@ pub mod ring_signature {
 // Re-export the one-time keys module which historically lived in this crate
 pub use mc_crypto_ring_signature::onetime_keys;
 
+// Re-export some dependent types from mc-account-keys
+pub use mc_account_keys::PublicAddress;
```

Comment:
This is definitely useful! Maybe also `AccountKey`?

Reply:
yah okay will do